src/enkf/gridio_fv3reg.f90

```diff
@@ -59,15 +59,15 @@ module gridio
                                 "T","W","DZ","delp"/) 
   character(len=max_varname_length), parameter :: vartracers(ntracerslist) =(/'sphum','o3mr', &
                                 'liq_wat','ice_wat','rainwat','snowwat','graupel','rain_nc'/)
-  type type_fv3lamfilebank 
+  type type_fv3lamfile 
        logical l_filecombined
        character(len=max_varname_length), dimension(2):: fv3lamfilename
        integer (i_kind), dimension(2):: fv3lam_fileid(2)
        contains
          procedure, pass(this) :: setupfilebank => type_bound_setupfilebank
```

Comment:
Thanks for making these changes.  Forgive my being picky. It will be good if we remove all "bank" suffixes,  setupfilebank  -> setupfile;  type_bound_setupfilebank  -> type_bound_setupfile.  Otherwise all looks good to me.

Reply:
@guoqing-noaa , Thanks.  All "filebank" strings have been replaced/reduced to "file".